modules/bq-log-alerting/logging/cloud_function/index.js

```diff
@@ -0,0 +1,165 @@
+/*
+Copyright 2020 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+const {BigQuery} = require('@google-cloud/bigquery');
+const {SecurityCenterClient} = require('@google-cloud/security-center')
+const MD5 = require('crypto-js/md5');
+
+const bigquery = new BigQuery();
+// These constants will need to be tuned to your preference
+const project = process.env.LOGGING_PROJECT;
+const dataset_name = 'views';
+const invocation_name = 'securityalert_invocation';
+const result_name = 'securityalert_result';
+const source_name = process.env.CSCC_SOURCE;
+const dry_run = (process.env.DRY_RUN === 'true');
+
+// Not currently used, was for original version of the cloud function
+getMetric = function(name, labels) {
```

Comment:
If these functions will not be used, we can remove them.

Reply:
removed